### PR TITLE
ci: add workflow to publish to npm

### LINKED ISSUE #8: Add github actions to publish automatically to npm
https://github.com/marketplace/actions/npm-publish

### PR DESCRIPTION
**Pull request by pyphilia**
close #8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,8 @@
   "peerDependencies": {
     "@material-ui/core": "4.11.0",
     "@material-ui/icons": "4.9.1",
-    "react": "^16.0.0"
+    "react": "^16.13.1",
```
Comment (juancarlosfarah) [inline package.json:39]: Should this be `^`? I forget what's needed for peerDeps.

File: .github/workflows/publishNpm.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
```
Comment (juancarlosfarah) [inline .github/workflows/publishNpm.yml:5]: Keep comments above, if possible and all small caps.

File: .github/workflows/publishNpm.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
```
Comment (juancarlosfarah) [inline .github/workflows/publishNpm.yml:3]: Minor thing: try to keep all comments lower case.

File: .github/workflows/publishNpm.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10
+      - run: yarn install
+      - run: yarn test
+
+      - name: Publish to NPM
```
Comment (juancarlosfarah) [inline .github/workflows/publishNpm.yml:18]: Minor thing. Let's all the names lowercase, if possible.

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Thanks for this! Just some very very minor comments and a question.

File: package.json (hunk 1 above)
Comment (pyphilia) [reply package.json:39]: I think that is okay to do this way. I checked material-ui and there's `^` in their peerDependencies https://github.com/mui-org/material-ui-pickers/blob/e875729b3492a4aedebec9f14589accc83475241/lib/package.json

**Comment by pyphilia** [comment]
Before merging, I will need a NPM_TOKEN. @juancarlosfarah  do we have a graasp account or should we use a token generated by one of us?

**Comment by juancarlosfarah** [comment]
@pyphilia we have an account. I believe you should be a part of it.

**Pull request merged**